main.c

```diff
@@ -1922,6 +1974,7 @@ int main(int argc, const char** argv)
         fprintf(flog,"Placement from file" ENDL);
     else
         fprintf(flog,"Tree depth: %d, cyber logic score: %d, cyber gameplay score: %d%s", treedepth, cyberlogicscore, cybergameplayscore, ENDL);
+    fprintf(flog,"Exp%%: %d, Money%%: %d%s", u8_fraction_to_percent(exp_num,exp_den), u8_fraction_to_percent(money_num,money_den), ENDL);
```

Comment:
you sure about `Money%: 100` instead of `Money: 100%`?

Reply:
tbh, I am fine with both. Just think `Money%: 100` is easy to spot in the log and matches the input fields.